### ISSUE TITLE
Manual update sys.path from current python interpreter, e.g. when using setuptools develop mode

### ISSUE THREAD

**Issue by kiryph** [davidhalter/jedi-vim#584]
setuptools provide installation variant for packages under development. This can be invoked in the following way:
```
$ python setup.py develop
```
For a comprehensive explanation see http://setuptools.readthedocs.io/en/latest/setuptools.html#develop-deploy-the-project-source-in-development-mode. Most importantly the files are only getting symlinked and `easy-install.pth` gets updated so that the project is on `sys.path`.

`$ python -c "import sys; print sys.path"` will return an additional entry for the project under development.

I propose an additional command for vim-jedi to allow completion for python projects under development: `JediUpdateSysPath`

**Comment by davidhalter** [comment]
And what would that command do? And why  would that `develop` package not be within the `sys.path`?

**Comment by kiryph** [comment]
I was suspecting that when I run vim, create a new python project which I put into development mode with `$ python setup.py develop`, jedi-vim won't notice that the sys.path got changed.
So the idea was that this command should read sys.path in again during a running vim instance.

Does this explain what I meant?

However, your comment on my second issue https://github.com/davidhalter/jedi-vim/issues/583#issuecomment-227013450 (jedi-vim uses the python interpreter built into vim) cleared things up.

Since I want to choose a different python interpreter compared to the one of vim, the idea of updating the sys.path for the completion is irrelevant. I do not install my projects under development into the system site-package location.

Anyhow, I switched to https://github.com/zchee/deoplete-jedi which has a variable to choose the interpreter `g:deoplete#sources#jedi#python_path`. I have to check whether I still would like to have a command to query the sys.path when running vim. But I can also live with a vim restart. It doesn't happen too often that the sys.path gets changed.

 If the issue is still unclear, simply ignore it.  I should have analyzed your plugin more thoroughly.
